metals/src/main/scala/scala/meta/internal/metals/MultilineStringFormattingProvider.scala

```diff
@@ -156,19 +181,21 @@ final class MultilineStringFormattingProvider(
     }
   }
 
-  def format(
-      params: DocumentRangeFormattingParams
-  ): Future[List[TextEdit]] = {
+  def format(params: DocumentRangeFormattingParams): Future[List[TextEdit]] = {
     val source = params.getTextDocument.getUri.toAbsolutePath
     val range = params.getRange()
     val doc = params.getTextDocument()
     withToken(doc, range, newlineAdded = false) { (sourceText, position) =>
-      val newText = indent(sourceText, position.start) + "|"
-      val lines = (range.getStart().getLine() + 1) to range.getEnd().getLine()
-      lines.map { line =>
-        val pos = new Position(line, 0)
-        new TextEdit(new Range(pos, pos), newText)
-      }.toList
+      val splitLines = sourceText.split('\n')
+      val defaultIndent = determineDefaultIndent(sourceText, position.start)
+      val linesToFormat = (range.getStart().getLine()) to range
```

Comment:
nit: try to avoid non-symbolic infix operators such as `to`

```suggestion
      val linesToFormat = range.getStart().getLine()).to(range.getEnd().getLine())
```

Reply:
Sounds good, made the change.